var/spack/repos/builtin/packages/filtlong/package.py

```diff
@@ -13,11 +13,14 @@ class Filtlong(MakefilePackage):
     homepage = "https://github.com/rrwick/Filtlong"
     url = "https://github.com/rrwick/Filtlong/archive/v0.2.0.tar.gz"
 
+    version("0.2.1", sha256="e6f47675e87f98cf2481a60bef5cad38396f1e4db653a5c1673139f37770273a")
     version("0.2.0", sha256="a4afb925d7ced8d083be12ca58911bb16d5348754e7c2f6431127138338ee02a")
     version("0.1.1", sha256="ddae7a5850efeb64424965a443540b1ced34286fbefad9230ab71f4af314081b")
 
     depends_on("zlib-api")
 
+    patch("gcc13.patch", level=0, when="%gcc@13:")
```

Comment:
Can you add a brief comment above the `patch` directive explaining what is the effect of the patch? That would help reading the recipe, since one would know what to expect in the patch without opening it.

Reply:
Done that for you @alalazo